src/Resource/Build.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace WyriHaximus\Travis\Resource;
```

Comment:
@WyriHaximus 

Not sure, is it even legal to use `Resource` as part of the namespace in PHP7?